Remove perfectly valid progress() test from invalid examples test

weinig opened the pull request:
#### d8dd547810630e20bc32e95706bb54d465f45949
<pre>
Remove perfectly valid progress() test from invalid examples test
<a href="https://bugs.webkit.org/show_bug.cgi?id=286560">https://bugs.webkit.org/show_bug.cgi?id=286560</a>

Reviewed by Anne van Kesteren and Tim Nguyen.

I accidentally left in a valid example of `progress()` in the
test for invalid ones.

* LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-invalid.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-invalid.tentative.html:

Canonical link: <a href="https://commits.webkit.org/289437@main">https://commits.webkit.org/289437@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/5f24ea91b9de0846ad5c12f2a0ff2451291faf14

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/86928 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/6435 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/41286 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/91776 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/37662 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/6703 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/14499 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/67201 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/24975 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/89931 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/5114 "Passed tests") | [❌ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/78699 "Failed to checkout and rebase branch from PR 39595") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/47519 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/4898 "Passed tests") | [❌ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/33072 "Failed to checkout and rebase branch from PR 39595") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/36778 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/75400 "Passed tests") | [❌ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/33951 "Failed to checkout and rebase branch from PR 39595") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/93669 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/14081 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/10234 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/75999 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/14282 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/74556 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/75196 "Passed tests") | 
| [  ~~🛠 🧪 merge~~](https://ews-build.webkit.org/#/builders/19/builds/18507 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/19522 "Passed tests") | [❌ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/17958 "Failed to checkout and rebase branch from PR 39595") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/6930 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/14104 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/13842 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/17287 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/15627 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
Should -computed/serialize variants of the test adopt this test case?

weinig committed:
Remove perfectly valid progress() test from invalid examples test
https://bugs.webkit.org/show_bug.cgi?id=286560

Reviewed by Anne van Kesteren and Tim Nguyen.

I accidentally left in a valid example of `progress()` in the
test for invalid ones.

* LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-invalid.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-invalid.tentative.html:

Canonical link: https://commits.webkit.org/289437@main
